src/amp/components/Epic.tsx

```diff
@@ -9,7 +9,7 @@ import {
 } from '@root/src/amp/components/moustache';
 import { palette } from '@guardian/src-foundations';
 import { headline, body, textSans } from '@guardian/src-foundations/typography';
-import { Ticker } from '@root/src/amp/components/Ticker';
+import { Ticker } from "@root/src/amp/components/Ticker";
```

Comment:
you have some prettier issues. A fix to force prettier to run on push has been added to main so maybe worth pulling from `main`

Reply:
Sorted 💅🏼